apps/intake/src/lib/i18n-en.json

```diff
@@ -496,6 +498,7 @@
     "Please provide the best mobile phone number for your family.": "Please provide the best mobile phone number for your family.",
     "Phone number": "Phone number",
     "s primary address": "'s primary address",
+    "Yes! I would like to receive helpful messages from {{PROJECT_NAME}} regarding patient education, events, and general information about our offices. Message frequency varies, and data rates may apply.": "Yes! I would like to receive helpful messages from {{PROJECT_NAME}} regarding patient education, events, and general information about our offices. Message frequency varies, and data rates may apply.",
```

Comment:
if this is a duplicate of like 242, why not place it in a common ancestor instead?

Reply:
sadly no, i'm unfamiliar. i can see the spanish file also has more additions and changes, so maybe there was a recent pr?